examples/tv-app/linux/include/media-playback/MediaPlaybackManager.cpp

```diff
@@ -141,6 +151,7 @@ void MediaPlaybackManager::HandleSeek(CommandResponseHelper<Commands::PlaybackRe
     if (positionMilliseconds > mDuration)
     {
         Commands::PlaybackResponse::Type response;
+        response.data   = chip::MakeOptional(CharSpan::fromCharString("data response"));
```

Comment:
These values don't really make sense.  Shouldn't we just be leaving `response.data` not set, if we have nothing useful to communicate?

Reply:
It's only for the testing purposes. As you know, the previous SDK was not supporting  char string as byte string was supported only. So to have test verification that this field is actually defined as string, that's why this random value is returned. 
The reasoning is that if null was used, there's no type verification.

Reply:
If that's wrong reasoning or there's a better way to achieve the verification, I'm all ears.

Reply:
@bzbarsky-apple